prepare

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env bash
+
+# get first part of version
+major=$(echo $npm_package_version | cut -d'.' -f1)
+
+# get second part of version
+minor=$(echo $npm_package_version | cut -d'.' -f2)
+
+# get third part of version
+patch=$(echo $npm_package_version | cut -d'.' -f3)
+
+# increment patch if first argument is "patch"
+if [ "$1" = "patch" ]; then
+  patch=$((patch + 1))
+fi
+
+# increment minor if first argument is "minor"
+if [ "$1" = "minor" ]; then
+  minor=$((minor + 1))
+  patch=0
+fi
+
+# increment major if first argument is "major"
+if [ "$1" = "major" ]; then
+  major=$((major + 1))
+  minor=0
+  patch=0
+fi
+
+# create new version
+new_version="$major.$minor.$patch"
+echo "Preparing $new_version"
+
+# Switch to new branch
+git checkout -b prepare-$new_version-release main 
```

Comment:
Just to be sure a GH main branch is used, not local one (in case it has any local changes).
Also it would be good to do a `git fetch` first?

```suggestion
git checkout -b prepare-$new_version-release origin/main 
```